libs/components/src/search/search.component.ts

```diff
@@ -30,7 +31,7 @@ let nextId = 0;
       useExisting: SearchComponent,
     },
   ],
-  imports: [InputModule, ReactiveFormsModule, FormsModule, I18nPipe],
+  imports: [InputModule, ReactiveFormsModule, FormsModule, I18nPipe, NgIf, NgClass],
 })
 export class SearchComponent implements ControlValueAccessor, FocusableElement {
```

Comment:
Nested forms is [not supported by the HTML spec](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/form#technical_summary). I did a spot-check in the code and I don't think we are using the search inside of a form right now, so we should be good. But I do want to make sure devs know about this when using the search component in the app code, so we can make a couple small updates to the Storybook docs.

Update 1:
Add a jsdoc comment to this component (doesn't have to be this exact text, I did not format this in a code editor either lol)
```suggestion
/**
 * Search components must not be nested inside of a form, since they contain their own standalone form for searching.
 */
export class SearchComponent implements ControlValueAccessor, FocusableElement {
```

Update 2:
In `search.mdx`, update line 1 to import "Description" too, then update line 11 to add:
`<Description />`
(you can check an existing mdx page like `icon-button.mdx` for an example) 
That should display the jsdoc comment on the Storybook page.

Reply:
I added a comment above the component and add `<Description />` to storybook. I ran storybook and confirmed that the description appears.